src/java.desktop/share/classes/java/awt/Window.java

```diff
@@ -3215,13 +3215,11 @@ public AccessibleStateSet getAccessibleStateSet() {
 
     @Override
     void setGraphicsConfiguration(GraphicsConfiguration gc) {
-        if (gc == null) {
-            gc = GraphicsEnvironment.
-                    getLocalGraphicsEnvironment().
-                    getDefaultScreenDevice().
-                    getDefaultConfiguration();
-        }
```

Comment:
It looks like `gc` was always not null before the fix, which is no longer true after the fix.
We need to double check that we didn't break anything in this regard.

Reply:
Yeah, have tested in CI multiple time and also few manual dual screen related test. No regression found with fix.

Reply:
We have a couple of places where we call "setGraphicsConfiguration(null)", previously that sets the default GC, now it will set the null, can we try to trigger that codepath and check how it will work?

Reply:
Removing a Component will call `Component.setGraphicsConfiguration()`, which in turn triggers `Component.updateGraphicsData()`. So `setGraphicsConfiguration(null)` never invoked `Window.setGraphicsConfiguration()`.

Reply:
Anyhow I have tested the updated code in CI system multiple time, never encountered a failure due to null. As part of precautionary measures, let me retain the old code too, along with updated ones.

Reply:
@mrserb I have addressed your review comments, Please let me know if anything else is there w.r.t PR? Let me know if the fix is fine.